src/diffusers/models/model_loading_utils.py

```diff
@@ -246,6 +246,17 @@ def load_model_dict_into_meta(
             else:
                 set_module_tensor_to_device(model, param_name, device, value=param)
 
+    for param_name, param in model.named_buffers():
```

Comment:
can we move the changes outside of the function? it takes a `state_dict` as input so it is strange that it also do something with model.named_buffer()

would it work if we run `load_model_dict_into_meta` on the buffers in a seperate call? if not, we could add an additional argument